docs/site/Binding.md

```diff
@@ -268,6 +394,29 @@ parameter of `BindingFromClassOptions` type with the following settings:
 - defaultScope: Default scope if the binding does not have an explicit scope
   set. The `scope` from `@bind` of the bound class takes precedence.
 
+{% include note.html content=" The `@bind` decorator only adds metadata to the
+class. It does NOT automatically bind the class to a context. To binding a class
+with `@bind` decoration, the following step needs to happen explicitly or
+implicitly (by a booter).
+
+```ts
+const binding = createBindingFromClass(AClassOrProviderWithBindDecoration);
+ctx.add(binding);
+```
+
+The metadata added by `@bind` is NOT inspected/honored by `toClass` or
+`toProvider`. Be warned that the example below does NOT set up the binding per
+`@bind` decoration:
+
+```ts
+const binding = ctx.bind('my-key').toClass(MyService);
+// The binding is NOT configured based on the `@bind` decoration on MyService.
+// The scope is default to BindingScope.TRANSIENT instead of BindingScope.SINGLETON.
```

Comment:
The state made earlier
```
The binding scope will impact values provided by `toDynamicValue`, `toClass`,
and `toProvider`.
...
By default, the binding scope is `TRANSIENT`.
```

Based on this statement, when is it ever singleton by default.

Perhaps instead of 
> // The scope is default to BindingScope.TRANSIENT instead of BindingScope.SINGLETON. 

we want to say:

// The scope is default to BindingScope.TRANSIENT